src/schedule.jl

```diff
@@ -267,6 +267,9 @@ function today(::Val{:terminal}; now, track, terminal_links)
     border_crayon = crayon"dark_gray"
     h_times = Highlighter((data, i, j) -> j == 1, crayon"white bold")
 
+    println()
+    println(TimeZones.Date(now))
```

Comment:
You're only printing the date in the `:terminal` output but not in `:text`. Is this intended? I think it's better if both output formats are as similar as possible.

Also, maybe we should use `Dates.format` to print the date in a nicer format? For example, "22 July, 2021" or something like that?